client/src/containers/PostDashboard.tsx

```diff
@@ -41,16 +51,28 @@ function RecentPosts(props: {
           })
           .catch((err) => console.log(err));
       }
-    }, 500);
+    }, 5000);
```

Comment:
This should be a separate PR? Additionally, if you increase polling then we should add something to instantly fetch posts when you create a post. Otherwise it will look like there is a 5 second lag when a post is made

Reply:
I think you already have that implemented, isn't that why you have 2 useEffects()?